docs/pages/setup/reference/backends.mdx

```diff
@@ -209,13 +209,14 @@ running on an EC2 instance with an IAM role.
 
 These optional `GET` parameters control how Teleport interacts with an S3 endpoint, including S3-compatible endpoints.
 
-`s3://bucket/path?region=us-east-1&endpoint=mys3.example.com&insecure=false&disablesse=false`
+`s3://bucket/path?region=us-east-1&endpoint=mys3.example.com&insecure=false&disablesse=false&acl=private`
```

Comment:
I think we can make it a bit more explicit that this URL is used to illustrate the arrangement of the params below. If you don't know the params are coming next, the role of the URL will take a bit of figuring out.